hpe_3par_kubernetes_manager.py

```diff
@@ -1042,10 +1042,10 @@ def verify_volume_properties(hpe3par_volume, **kwargs):
         if 'copyOf' in kwargs:
             if hpe3par_volume['copyOf'] != kwargs['copyOf']:
                 return False
-        # For Arcus        
-        # if 'snapCPG' in kwargs:
-        #     if hpe3par_volume['snapCPG'] != kwargs['snapCPG']:
-        #         return False
+        if 'snapCPG' in kwargs:
+            if int(globals.hpe3par_version[0:1]) >= 3:

```

Comment:
Verify the platform type instead of parsing the version. This way you never have modify the code again.

Reply:
Resolved in newest commit